go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rhysd/actionlint
 
-go 1.18
+go 1.23.5
```

Comment:
I overlooked this line. This line is not acceptable. I'll remove this change from main branch.

https://github.com/rhysd/actionlint/blob/main/docs/api.md#go-version-compatibility